main.py

```diff
@@ -0,0 +1,19 @@
+# Download the helper library from https://www.twilio.com/docs/python/install
+import os
+from twilio.rest import Client
+
+
+# Find your Account SID and Auth Token at twilio.com/console
+# and set the environment variables. See http://twil.io/secure
+account_sid = os.environ['ACCOUNT_SID']
+auth_token = os.environ['AUTH_TOKEN']
+client = Client(account_sid, auth_token)
+
+message = client.messages \
+                .create(
+                     body="Apparently, we are pencils. Like your *redacted.",
+                     from_='+14845099095',
+                     to='+4915222072245'
+                 )
+
+print(message.sid)
```

Comment:
Why are you printing this message? I'll assume you should log messages sent in a DB so we can see them later